osquery/tables/system/posix/authorized_keys.cpp

```diff
@@ -7,25 +7,45 @@
  * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
  */
 
-#include <string>
 #include <vector>
 
 #include <osquery/core/core.h>
 #include <osquery/core/tables.h>
 #include <osquery/filesystem/filesystem.h>
-#include <osquery/logger/logger.h>
+
+#include <osquery/tables/system/posix/authorized_keys.h>
 #include <osquery/tables/system/system_utils.h>
+#include <osquery/utils/conversions/join.h>
 #include <osquery/utils/conversions/split.h>
 #include <osquery/utils/system/system.h>
 #include <osquery/worker/ipc/platform_table_container_ipc.h>
-#include <osquery/worker/logging/glog/glog_logger.h>
 
 namespace osquery {
 namespace tables {
 
 const std::vector<std::string> kSSHAuthorizedkeys = {".ssh/authorized_keys",
                                                      ".ssh/authorized_keys2"};
 
+const std::vector<std::string> kSSHKeyTypes = {"ssh-rsa",
+                                               "ssh-ed25519",
+                                               "ecdsa-sha2-nistp256",
+                                               "ecdsa-sha2-nistp384",
+                                               "ecdsa-sha2-nistp521"};
+
+const std::string kKeyRingLabelOptionPrefix = "zos-key-ring-label=";
```

Comment:
What uses this?

Reply:
I've moved it to the function that checks the existence of the 'zos-key-ring-label' parameter in the options section instead of the global constant.
This option changes the line syntax. If present the line should look: `from="*.example.com",zos-key-ring-label="KeyRingOwner"` 
Please notice that the following parameters key type, key, and comment shouldn't be specified.